test/integration/ari_test.go

```diff
@@ -45,28 +44,38 @@ func TestARI(t *testing.T) {
 	test.AssertNotError(t, err, "failed to issue test cert")
 
 	cert := ir.certs[0]
-	issuer, err := ocsp_helper.GetIssuer(cert)
-	test.AssertNotError(t, err, "failed to get issuer cert")
-
-	eri, err := client.GetRenewalInfo(cert, issuer, crypto.SHA256)
+	ari, err := client.GetRenewalInfo(cert)
 	test.AssertNotError(t, err, "ARI request should have succeeded")
-	test.AssertEquals(t, eri.SuggestedWindow.Start.Sub(time.Now()).Round(time.Hour), 1415*time.Hour)
-	test.AssertEquals(t, eri.SuggestedWindow.End.Sub(time.Now()).Round(time.Hour), 1463*time.Hour)
-	test.AssertEquals(t, eri.RetryAfter.Sub(time.Now()).Round(time.Hour), 6*time.Hour)
+	test.AssertEquals(t, ari.SuggestedWindow.Start.Sub(time.Now()).Round(time.Hour), 1415*time.Hour)
+	test.AssertEquals(t, ari.SuggestedWindow.End.Sub(time.Now()).Round(time.Hour), 1463*time.Hour)
+	test.AssertEquals(t, ari.RetryAfter.Sub(time.Now()).Round(time.Hour), 6*time.Hour)
+
+	// TODO(@pgporada): Clean this up when 'test/config/{sa,wfe2}.json' sets
+	// TrackReplacementCertificatesARI=true.
+	if os.Getenv("BOULDER_CONFIG_DIR") == "test/config-next" {
+		ir, err = authAndIssueReplacement(client, key, []string{name}, true, cert)
+		test.AssertNotError(t, err, "failed to issue test cert")
+		test.AssertNotEquals(t, ir.Replaces, "")
+
+		// Try it again and verify it fails
+		_, err = authAndIssueReplacement(client, key, []string{name}, true, cert)
+		test.AssertError(t, err, "subsequent ARI replacements for a replaced cert should fail, but didn't")
```

Comment:
If the only thing you're testing here is "can I make an order with a replaces field" and "can I _not_ make a second order with a replaces field", then I think you could accomplish that goal just with `makeClientAndOrder`? The returned order object in the first step should have a populated "replaces" field (which you should check for the expected equality, not just that it's not ""), and the second step should error out.

This would remove the need for a new `authAndIssueReplacement` helper.